airqo-platform/AirQo-api
# changing file perms the entry bash script  and CLI logs for node micros

# Hotfix

**_WHAT DOES THIS PR DO?_**

- [x] Changes the file permissions for the entry bash script of all nodeJS microservices. This ensures that one can run `npm start` successfully. 

- [x] When running npm start, same PR also changes the console log output for the development environment from "undefined" to "production". This is just an enhancement since "undefined" environment was always defaulting to production environment in the code implementation.

**_WHAT JIRA STORY/TASK IS RELATED TO THIS PR?_**
[PLAT-305](https://airqoteam.atlassian.net/browse/PLAT-305?atlOrigin=eyJpIjoiYzc3ODk2MTdmMjkwNDlkMTg5YTc4MmVkYzNjZjFhZDUiLCJwIjoiaiJ9)

**_WHAT IS THE LINK TO THE PR BRANCH_**
https://github.com/airqo-platform/AirQo-api/tree/hotfix-chmod-www

**_HOW DO I TEST OUT THIS PR?_**

> Data Management

cd AirQo-api/src/data-mgt/node
`npm install`
`npm start`
The observe the console log statements in your command line. It should show _Running on production_

> Device Registry

`cd AirQo-api/src/device-registry`
`npm install`
`npm start`
The observe the console log statements in your command line. It should show _Running on production_

> Auth Service

`cd AirQo-api/src/auth-service`
`npm install`
`npm start`
The observe the console log statements in your command line. It should show _Running on production_

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 
N/A

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**
N/A

**_ARE THERE ANY RELATED PRs?_**
N/A
